theano/gof/cmodule.py

```diff
@@ -2009,11 +2009,12 @@ def join_options(init_part):
         # in the key of the compiled module, avoiding potential conflicts.
 
         # Figure out whether the current Python executable is 32
-        # or 64 bit and compile accordingly. This step is ignored for ARM
-        # architectures in order to make Theano compatible with the Raspberry
-        # Pi, and Raspberry Pi 2.
+        # or 64 bit and compile accordingly. This step is ignored for 
```

Comment:
The tests are failing as there are white spaces at the end of a few lines. Just remove the extra space.